Fix build errors like the following by pointing tsc(1) to a local copy of
the "ms" module under debian/xtradeb-fix/:

| Found debian/nodejs/llparse-builder/build
|	cd ./llparse-builder && sh -ex ../debian/nodejs/llparse-builder/build
| + tsc
| ../../../usr/share/nodejs/@types/debug/index.d.ts:27:33 - error TS2307: Cannot find module 'ms' or its corresponding type declarations.
|
| 27         humanize: typeof import('ms');
|                                    ~~~~
|
| Found 1 error.

(Note: Installing the "node-ms" package does not resolve the issue, partly
because the version is too old)

--- a/llhttp/tsconfig.json
+++ b/llhttp/tsconfig.json
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "../debian/xtradeb-fix",
     "strict": true,
     "target": "es2017",
     "module": "commonjs",
--- a/llparse-builder/tsconfig.json
+++ b/llparse-builder/tsconfig.json
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "../debian/xtradeb-fix",
     "strict": true,
     "target": "es2017",
     "module": "commonjs",
--- a/llparse-frontend/tsconfig.json
+++ b/llparse-frontend/tsconfig.json
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "../debian/xtradeb-fix",
     "strict": true,
     "target": "es2017",
     "module": "commonjs",
--- a/llparse/tsconfig.json
+++ b/llparse/tsconfig.json
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "../debian/xtradeb-fix",
     "strict": true,
     "target": "es2017",
     "module": "commonjs",
